### PR TITLE
Assignment 1 and Assignment 2 review

### PR DESCRIPTION
**Pull request by snehil1930**

### CODE REVIEW

File: test/Assig1/ValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    private ValidatorUtils validator;
+
+    /*
+     * This test method is used to check whether correct input is passing test or not
+     */
+    @Test
+    void devTest() throws Exception {
+
+        String[] args = new String[]{"-name", "snehil", "-type", "raw", "-price", "123", "-quantity", "2"};
+        validator = new ValidatorUtils();
```
Comment (amangoyal22) [inline test/Assig1/ValidatorTest.java]: util class ? why are u declaring it again

File: test/Assig1/ValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
+package Assig1;
```
Comment (amangoyal22) [inline test/Assig1/ValidatorTest.java]: package naming

File: test/Assig1/ValidatorTest.java [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+     * it test the final price of raw item after calculation
+     */
+    @Test
+    void finalPriceRawTest() throws Exception {
+        validator = new ValidatorUtils();
+        String[] args = new String[]{"-name", "snehil", "-type", "raw", "-price", "123", "-quantity", "2"};
+        Item it = validator.getDetails(args);
+        it.calculateFinalPrice();
+        float expected = (float) 276.75;
+        assertEquals(it.getFinalPrice(), expected);
```
Comment (amangoyal22) [inline test/Assig1/ValidatorTest.java]: validator Class should have validator test case no item class test case

**Review by amangoyal22 — CHANGES_REQUESTED** [review]
Assignment 1 ->
- Package naming is not proper.
- Test case of individual classes is missing
- ValidatorUtils is final class, should not have new instance need everytime

File: test/Assig1/ValidatorTest.java (hunk 1 above)
Comment (snehil1930) [reply test/Assig1/ValidatorTest.java]: ok

File: test/Assig1/ValidatorTest.java (hunk 2 above)
Comment (snehil1930) [reply test/Assig1/ValidatorTest.java]: corrected

File: test/Assig1/ValidatorTest.java (hunk 3 above)
Comment (snehil1930) [reply test/Assig1/ValidatorTest.java]: made separte test case

**Pull request merged**